project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
     // Scala
     val collectorPayload = "0.0.0"
     val tracker          = "1.0.0"
-    val akkaHttp         = "10.2.6"
+    val akkaHttp         = "10.2.7"
```

Comment:
Just to double-check: the max parsing depth will be configurable using the `akka` section of the config HOCON, correct?

Reply:
It will be configurable, in the same way that all akka options are configurable in the `akka` seciont.  But we will leave it on the akka default, which is 5.  I don't see a reason to change from the default.